docs/topics/build-and-deploy-process.adoc

```diff
@@ -1,6 +1,7 @@
 [[build-and-deploy-process]]
-= Building and Deploying to OpenShift Build Process
+= Build and Deploy to OpenShift
```

Comment:
Please keep the gerund form.

Reply:
I am perfectly happy with that reasoning, Plus, it avoids the awful long title. Fixing...